src/components/mdx-img/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface MdxImgProps {
```

Comment:
Could you update this so the docs comments are always multi-line (like `Button`, as a recent example) and also put an empty line between each property for improved readability?